docs/Getting-Started/Private-Network-Quickstart.md

```diff
@@ -410,61 +431,66 @@ Compiling truffle/DeployedAddresses.sol...
 
-Continue with the regular tutorial steps from the [Installing and configuring lite-server section](https://truffleframework.com/tutorials/pet-shop#installing-and-configuring-lite-server) to the end of the tutorial.
+Continue with the regular tutorial steps from the [Installing and configuring lite-server](https://truffleframework.com/tutorials/pet-shop#installing-and-configuring-lite-server)
+section and finish the tutorial.
 
-When you adopt pets in the browser and approve the transaction in MetaMask, you will be able to see the transactions in the block explorer. 
+When you adopt pets in the browser and approve the transaction in MetaMask, you'll be able to see the transactions in the block explorer.
 
-## Shut Down the Network and Remove the Containers
- 
-To shut down the network and delete all containers:
 
-```bash tab="Linux/macOS"
-# Shell script are provided in the Quickstart directory
+## Stop / Restart Private Network without Removing Containers 
 
-quickstart/removePantheonPrivateNetwork.sh
-```
+To shut down the private network without deleting the containers:
 
-```bat tab="Windows"
-// Run the docker-compose commands directly
+```bash tab="Linux/MacOS"
+# Stop the containers:
 
-quickstart\docker-compose down
+./stop.sh
 ```
 
-!!!note
-    On Windows, the quickstart creates a volume called `quickstart_public-keys` but it's not automatically removed. Remove this volume using `docker volume rm quickstart_public-keys`.
+```bash tab="Windows"
+// Stop the containers:
 
-## Stop and restart the Private Network without Removing the Containers 
+.\stop.sh
+```
+
+!!! note
+    This command will also stop other running Docker containers unrelated to the Pantheon quickstart.
 
-To shut down the network without deleting the containers:
+To restart the private network:
 
-```bash tab="Linux/macOS"
-# Shell script are provided in the Quickstart directory
+```bash tab="Linux/MacOS"
+# Start the containers:
 
-quickstart/stopPantheonPrivateNetwork.sh
+./start.sh
 ```
 
-```bat tab="Windows"
-// Run the docker-compose commands directly
+```bash tab="Windows"
+// Start the containers:
 
-quickstart\docker-compose stop
+.\start.sh
 ```
 
-(This command will also stop other running containers unrelated to quickstart.)
 
-To restart the private network:
+## Stop Private Network and Remove Containers
+
+To shut down the private network and delete all containers:
 
-```bash tab="Linux/macOS"
-# Shell script are provided in the Quickstart directory
+```bash tab="Linux/MacOS"
+# Stop and delete the containers:
 
-quickstart/resumePantheonPrivateNetwork.sh
+./remove.sh
 ```
 
-```bat tab="Windows"
-// Run the docker-compose commands directly
+```bash tab="Windows"
+// Stop and delete the containers:
+
+.\remove.sh
+```
 
-quickstart\docker-compose start
-```
\ No newline at end of file
+!!! info "Windows"
```

Comment:
It should be. I can add this step in the remove script.
I created the related issue : https://github.com/PegaSysEng/pantheon-quickstart/issues/10